implementations/rust/ockam/ockam_command/src/share/accept.rs

```diff
@@ -0,0 +1,85 @@
+use clap::Args;
+use miette::IntoDiagnostic;
+use tokio::sync::Mutex;
+use tokio::try_join;
+use tracing::debug;
+
+use ockam::Context;
+use ockam_api::cloud::share::{AcceptInvite, AcceptedInvite};
+
+use crate::node::util::delete_embedded_node;
+use crate::util::api::{self, CloudOpts};
+use crate::util::{node_rpc, Rpc};
+use crate::{docs, CommandGlobalOpts};
+
+const PREVIEW_TAG: &str = include_str!("../static/preview_tag.txt");
+
+#[derive(Clone, Debug, Args)]
+#[command(
+    before_help = docs::before_help(PREVIEW_TAG),
+)]
+pub struct AcceptCommand {
```

Comment:
I am questioning the very title of this PR here :-). Do we need to implement the command right away? Can we just hook the invitation flow to the desktop app for now? I think that we can create separate github issues if people want to extend the ockam command. 

If this is for testing I'd rather write some integration tests. For example, for the enroll functionality I added [those tests](https://github.com/build-trust/ockam/blob/9171d74d4ad4ab66658dce7c9123acfb2e0632ff/implementations/rust/ockam/ockam_command/src/enroll/oidc_service.rs#L469). Unfortunately a few of them open the browser so I had to mark them as ignored. But your tests should be able to run without having to run bats tests.

Reply:
I don't see harm in doing this work now as it's not a significant amount of added code and little of it is superfluous from the desktop POV. I still need an acceptance flow in both cases and my feedback loop for vetting the end-to-end implementation can be considerably easier with CLI vs Tauri.